index.js

```diff
@@ -65,6 +65,20 @@ export default class QRCodeScanner extends Component {
         </Text>
       </View>
     ),
+    pendingAuthorizationView: (
+      <View style={{
+        flex: 1,
+        alignItems: 'center',
+        justifyContent: 'center',
+      }}>
+        <Text style={{
+          textAlign: 'center',
+          fontSize: 16,
+        }}>
+          ...
+        </Text>
+      </View>
+    ),
```

Comment:
wondering if it's a good idea to have some sort of a spinner or animated indicator here.

Reply:
Good idea!